_posts/2020-01-13-release-r118/index.md

```diff
@@ -0,0 +1,216 @@
+
+In addition, this release also includes important refactoring in the libraries that we use (e.g. `cats` instead of `scalaz` and `circe` instead of`json4s`),
+a bump of [Beam](https://beam.apache.org/) version to 2.11.0,
+an [updated](https://snowplowanalytics.com/blog/2019/08/09/iglu-scala-client-0.6.0-released/) [Iglu Client](https://github.com/snowplow/iglu-scala-client)
+with different validator and fixed caching mechanism,
+as well as an improvement to the referer parser enrichment.
+
+[Scala Stream Collector](https://github.com/snowplow/snowplow/tree/master/2-collectors/scala-stream-collector)
+and the enrich jobs versions ([Stream Enrich](https://github.com/snowplow/snowplow/tree/master/3-enrich/stream-enrich) and
+[Beam Enrich](https://github.com/snowplow/snowplow/tree/master/3-enrich/beam-enrich)) are bumped to `1.0.0`.
+
+As introduced in the [RFC](https://discourse.snowplowanalytics.com/t/rfc-making-the-snowplow-pipeline-real-time-end-to-end-and-deprecating-support-for-batch-processing-modules/3018),
+we stop supporting Spark from this release, in favor of streaming.
+
+1. [Beta release](#betarelease)
+2. [New bad row format](#badrows)
+3. [Referer parser enrichment improvement](#refererparser)
+4. [Upgrading](#upgrading)
+5. [Roadmap](#roadmap)
+6. [Getting help](#help)
+
+<h1 id="betarelease">1. Beta release</h1>
+
+This release represents a major upgrade of the core Snowplow technology. We are currently testing this tech internally - no small feat because of the size and scope of the release. Whilst we are running those tests, we have decided to make the code available to open source users to test themselves: we would welcome any feedback on the new functionality, which we hope to release in production in [R119](#roadmap).
+
+If you are an open source users who would like to test this new functionality, we would be happy to provide a high-priority OSS support during this testing phase (covering setup, debug, recovery or other aspects). We are committed to iterating our release candidates for R119 rapidly.
+
+<h1 id="badrows">2. New bad row format</h1>
+
+<h2>2.1. Why update the bad row format?</h2>
+
+The Snowplow pipeline has always been non-lossy: when there is an issue processing an event, rather than silently discard the event, it is saved as a "bad row" - opening up the possibility to monitor the number of bad rows, inspect them, fix them and reprocess them.
```

Comment:
```suggestion
The Snowplow pipeline has always been non-lossy: when there is an issue processing an event, rather than silently discard the event, it is saved as a "bad row" . This is only half the challenge! Snowplow also gives the ability to monitor the number of bad rows, inspect them, fix them and reprocess them to complete your data set.
```